### PR TITLE
Resolves #2140 NOTICES - MFIP ORIENTATION Address list

### LINKED ISSUE #2140: NOTICES - MFIP ORIENTATION needs address list added
Currently pulls address info from Global Variables, which is no longer supported.  An address list will be added to the script itself similar to current Appointment letter script.  I have this.

### PR DESCRIPTION
**Pull request by courtrightd**
Blip: Added a list of agency office addresses within the script to correct an issue with the address auto-fill in the dialog.

### CODE REVIEW

File: Script Files/NOTICES/NOTICES - MFIP ORIENTATION.vbs [hunk 1]
```diff
@@ -44,76 +48,775 @@ IF IsEmpty(FuncLib_URL) = TRUE THEN	'Shouldn't load FuncLib if it already loaded
...
+agency_address.state = "MN"
+
 'THE SCRIPT----------------------------------------------------------------------------------------------------
 'Connects to BlueZone
 EMConnect ""
 
 'Searches for a case number
 call MAXIS_case_number_finder(case_number)
-
+err_msg = ""
```
Comment (theVKC) [inline Script Files/NOTICES/NOTICES - MFIP ORIENTATION.vbs]: Is this one necessary @courtrightd?

File: Script Files/NOTICES/NOTICES - MFIP ORIENTATION.vbs [hunk 2]
```diff
@@ -44,76 +48,775 @@ IF IsEmpty(FuncLib_URL) = TRUE THEN	'Shouldn't load FuncLib if it already loaded
...
+
+county_office_list = ""     'Blanking this out because it may contain old info from the old global variables (from before this was integrated in this script)
+
+call convert_array_to_droplist_items(agency_office_array, county_office_list)
 
 'DIALOGS----------------------------------------------------------------------------------------------------
 'Must modify county_office_list manually each time to force recognition of variable from functions file. In other words, remove it from quotes.
-BeginDialog MFIP_orientation_dialog, 0, 0, 366, 155, "MFIP orientation letter"
+BeginDialog Dialog1, 0, 0, 366, 155, "MFIP orientation letter"
   EditBox 60, 5, 55, 15, case_number
```
Comment (theVKC) [inline Script Files/NOTICES/NOTICES - MFIP ORIENTATION.vbs]: Can you update this to read `MAXIS_case_number` across the script before I accept it, @courtrightd?

Comment (courtrightd) [inline Script Files/NOTICES/NOTICES - MFIP ORIENTATION.vbs]: Done.

File: Script Files/NOTICES/NOTICES - MFIP ORIENTATION.vbs [hunk 3]
```diff
@@ -168,3 +870,10 @@ call write_variable_in_case_note("---")
 call write_variable_in_case_note(worker_signature)
 
 script_end_procedure("")	'Script ends
+eduled on: " & orientation_date & " at " & orientation_time)
```
Comment (theVKC) [inline Script Files/NOTICES/NOTICES - MFIP ORIENTATION.vbs]: This is odd...

**Comment by courtrightd** [comment]
@theVKC sorry for the sloppy work, should have cleaned this up better yesterday.  Your comments and a couple of my own issues are updated now and I retested the new version.

**Comment by theVKC** [comment]
And now, we have a merge conflict.

@courtrightd, can you sync your branch with mine and correct any issues?

**Comment by courtrightd** [comment]
I'm having issues and don't have time for learning today.  I'm going to close this and repull from a new fork.

**Comment by theVKC** [comment]
This is what happens when I update 180 scripts in a single day. I'll be doing it again as soon as I get your request in. I'm waiting on a mass change. :)

**Pull request closed without merging**